pkg/controller/podautoscaler/legacy_horizontal_test.go

```diff
@@ -753,111 +752,6 @@ func TestLegacyScaleDownIgnoresUnreadyPods(t *testing.T) {
 	tc.runTest(t)
 }
 
-func LegacyTestTolerance(t *testing.T) {
```

Comment:
@kubernetes/sig-autoscaling-misc should these be removed or promoted back to active tests?

Reply:
Okay. Let me try to promote them back to active tests. Thanks for your feedback. :)